esp/templates/program/modules/teacherclassregmodule/listclasses.html

```diff
@@ -130,7 +130,7 @@ <h3>Cancellation Request for {{ cls|truncatewords:6 }}</h3>
     </a>
     <br />
     <a class="create-class-link" href="/teach/{{ one }}/{{ two }}/copyclasses" title="Teach your existing class for {{ settings.ORGANIZATION_SHORT_NAME }}!">
-      Teach a class from a previous program for {{ program.niceName }}!
+      Copy a previously registered class (including from a previous program) for {{ program.niceName }}!
```

Comment:
```suggestion
      Copy a registered class (including from a previous program) for {{ program.niceName }}!
```

I think it's clearer without the "previously"